packages/Disclosure/src/disclosure.scss

```diff
@@ -213,14 +213,13 @@
     }
 
     .ids-disclosure__header-content {
-        display: block;
         margin: var(--ids-disclosure-header-content-margin);
+        display: flex;
+        flex: 1 1 auto;
 
         /* stylelint-disable-next-line media-query-no-invalid */
         @media (width >= #{tokens.$breakpoints-sm}) {
             align-items: baseline;
-            display: flex;
```

Comment:
this should stay here and only flex: 1 1 auto should be moved out of the breakpoint.